### ISSUE TITLE
Ignore unparsable stats

### ISSUE THREAD

**Issue by candicenutsfit** [Nstalgic/Artemis#1]
For some reason, there are some files that the game outputs that are corrupted. I have narrowed it down to a few corrupted files and a couple of 0kb ones as well
![image](https://user-images.githubusercontent.com/86062489/122412046-d554c780-cfb7-11eb-8e93-937288030ebe.png)

Current behaviour: Program gets stuck when attempting to parse invalid files

Expected behaviour: Skip over broken files and possibly show the user which files were affected

**Comment by Nstalgic** [comment]
That's amazing... This has been fixed and will be included in the next update. Thanks for the report!

**Comment by candicenutsfit** [comment]
Tested the update. I have found that on first load (before saved_data is created), the broken file causes the rest of the same name scenarios to fail to load.

For example, 1w4ts Voltaic has a single broken file on initial load. The result is that the scenario has "No Data": 
![image](https://user-images.githubusercontent.com/86062489/122451836-2a560500-cfdb-11eb-9cf1-e6464149bdbe.png)

After further testing, I found that if I added the broken file after loading the clean dataset, only that file fails to load.

Not familiar with the codebase but this uncaught exception may be part of the problem.
https://github.com/Nstalgic/Artemis/blob/81d28f0b860c5982546f00b768e88fd30f9a7148/main.py#L681
